packages/web/src/components/notification/Notifications/MilestoneNotification.tsx

```diff
@@ -19,13 +24,54 @@ const messages = {
   reached: 'has reached over'
 }
 
+export const formatAchievementText = (
+  type: string,
+  name: string,
+  value: number,
+  achievement: string
+) => {
+  const achievementText =
+    achievement === Achievement.Listens ? 'Plays' : achievement
```

Comment:
not a CR comment but - I see this line all over Twitter, so I know it's old code, but it lowkey bothers me that "Plays" is capitalized here